src/tools/idlc/src/types.c

```diff
@@ -424,6 +479,51 @@ emit_enum(
   return IDL_VISIT_DONT_RECURSE;
 }
 
+static idl_retcode_t
+emit_bitmask(
+  const idl_pstate_t *pstate,
+  bool revisit,
+  const idl_path_t *path,
+  const void *node,
+  void *user_data)
+{
+  struct generator *gen = user_data;
+  char *name = NULL, *type = NULL;
+  const char *fmt, *base_type_str;
+  const idl_bitmask_t *bitmask = (const idl_bitmask_t *)node;
+  const idl_bit_value_t *bit_value;
+
+  (void)pstate;
+  (void)revisit;
+  (void)path;
+  if (IDL_PRINTA(&type, print_type, node) < 0)
+    return IDL_RETCODE_NO_MEMORY;
+  assert(bitmask->bit_bound > 0 && bitmask->bit_bound <= 64);
```

Comment:
This is checking if the parser did it's job correctly... Probably best to assume the parser is correct and the resulting tree is too(?)

Reply:
Fixed in c80f50d